### DISCUSSION TITLE
Strongswan High Availability - Link Failure Scenario

### DISCUSSION THREAD

**Discussion by NunzioMS** [strongswan/strongswan#997] [Q&A]
Hi,

As per your main presentation of the Strongswan HA Solution https://docs.strongswan.org/strongswan-docs/5.9/features/highAvailability.html it looks that only these potential scenario can be detected as failure and trigger the node Failover.

![image](https://user-images.githubusercontent.com/103967090/163882516-512a38d3-2c35-4023-9252-dc6e24e965c4.png)

To confirm before patching my kernel, this solution is not able to detect any Internal / External link failure of one of the node and switchover to the remaining node?

Thanks for your help,

**Comment by tobiasbrunner** [comment]
> To confirm before patching my kernel, this solution is not able to detect any Internal / External link failure of one of the node and switchover to the remaining node?

No, you'd have to script this.
